Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm

```diff
@@ -359,18 +361,24 @@ static inline uint8_t authDataFlags(ClientDataType type, LocalConnection::UserVe
     // Step 4.
     if (largeBlobInput->write) {
         auto nsCredentialId = toNSData(response->rawId());
-        NSDictionary *fetchQuery = @{
+        NSMutableDictionary *fetchQuery = [@{
             (id)kSecClass: (id)kSecClassKey,
             (id)kSecAttrKeyClass: (id)kSecAttrKeyClassPrivate,
             (id)kSecAttrSynchronizable: (id)kSecAttrSynchronizableAny,
-            (id)kSecAttrApplicationLabel: nsCredentialId.get(),
+            (id)kSecAttrAlias: nsCredentialId.get(),
             (id)kSecUseDataProtectionKeychain: @YES,
             (id)kSecReturnAttributes: @YES,
             (id)kSecReturnPersistentRef: @YES,
-        };
+        } mutableCopy];
```

Comment:
This is a memory leak, since `mutableCopy` returns a +1 object. Please use `RetainPtr` and `adoptNS`.

Reply:
I've used the `adoptNS` - `setDictionary` pattern I saw elsewhere in this code.